modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Color.js

```diff
@@ -0,0 +1,5 @@
+// imports the tiptap extension from node_modules
```

Comment:
Interesting that we do this but don't extend it in any way, there are tiptap extensions we just import directly I think, we don't map one to one to `tiptap-extensions` for everything. But this provides a nice point for future extension.